docs/console/usage-billing.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: usage-billing
+title: Usage-based billing
+sidebar_label: Usage-based billing
+slug: usage-billing
+---
+
+# Usage-based billing
+
+:::info
+
+This feature is in preview. While we are rolling out usage-based billing, we will not collect payment for overusage. Payment
+collection will start XX.XX.2024.
+
+Please report any issues to support@ory.sh or contact us in the [Ory Community](https://slack.ory.sh/).
+
+:::
+
+## Overview
+
+Usage-based invoices is a new feature in the Ory Console that you to understand your usage of Ory Cloud services, the usage that
+is included in your plan, and the overuse. Usage-based invoices are generated on a monthly basis.
```

Comment:
Maybe also remove the "new", as it's not going to be new in a couple of months, but we're probably not going to remove it here.

> Usage-based invoices are generated every month.

Are they really? I thought we still generate the month-to-date invoice with partial data?

Reply:
Yes, but the basis for the calculation is the current usage period, which is a month.

Reply:
I reworded, PTAL :)